### PR TITLE
Fix loc strings.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -1630,9 +1630,9 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile",
-    "compile": "yarn run precompile && webpack --mode production --vscode-nls",
+    "compile": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls",
     "compile-dev": "yarn run precompile && webpack --mode development",
-    "compile-watch": "yarn run precompile && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
+    "compile-watch": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
     "compile-dev-watch": "yarn run precompile && webpack --mode development --watch --info-verbosity verbose",
```
Comment (sean-mcmanus) [inline Extension/package.json:1630]: This could also go in precompile (not sure what is preferred).

Comment (Colengms) [reply Extension/package.json:1630]: I don't think this is a good idea.  compile-watch is not for a production build, so shouldn't incur the overhead of generating translations.  If you need translations, use the production build.

Comment (Colengms) [reply Extension/package.json:1630]: Er, it looks like it's needed on compile, but I'd prefer to don't add it to compile-watch.

Comment (sean-mcmanus) [reply Extension/package.json:1630]: Ok

File: Extension/package.json [hunk 2]
```diff
@@ -1624,7 +1624,7 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile",
-    "compile": "yarn run precompile && webpack --mode production --vscode-nls",
+    "compile": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls",
     "compile-dev": "yarn run precompile && webpack --mode development",
     "compile-watch": "yarn run precompile && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
```
Comment (Colengms) [inline Extension/package.json]: Sorry, I gave you bad feedback on this.  Obviously, 'compile-watch' is for production and 'compile-dev-watch' is not.  Could you put the 'yarn run translations-generate' back on 'compile-watch'.   :]

Comment (sean-mcmanus) [reply Extension/package.json]: Sure

**Review by Colengms — APPROVED** [review]

**Pull request merged**